priv/templates/bin

```diff
@@ -69,16 +69,25 @@ ERTS_LIB_DIR="$ERTS_DIR/../lib"
 [ -f "$REL_DIR/$REL_NAME.boot" ] && BOOTFILE="$REL_NAME" || BOOTFILE=start
 cd "$ROOTDIR"
 
+{{! Define alternative field separator using ASCII US (unit separator). US is
+    not expected to appear in any argument. }}
+IFS_NORM="$IFS"
+IFS_ARGS="$(printf '\x1f')"
+
 # Save extra arguments
-ARGS=$*
+{{! Join extra arguments into a US-delimited string. }}
+IFS="$IFS_ARGS"
+ARGS="$*"
+IFS="$IFS_NORM"
 
 # Build arguments for erlexec
-set -- "$ERL_OPTS"
+set --
+[ "$ERL_OPTS" ] && set -- "$@" "$ERL_OPTS"
```

Comment:
Technically the `"$@"` is not needed here, as it's empty at this point, but it can be there for consistency with the following lines which further build up the argument list.